backend/src/main/java/com/redhat/cloud/notifications/routers/UserConfigResource.java

```diff
@@ -132,6 +135,7 @@ public Response saveSettingsByEventType(@Context SecurityContext sec, @NotNull @
     @Produces(APPLICATION_JSON)
     @Tag(name = OApiFilter.PRIVATE)
     public Response getSettingsSchemaByEventType(@Context SecurityContext sec) {
+        forbidAccessInCaseOfServiceAccountAuthentication(sec);
```

Comment:
Shouldn't the other `GET` method (line 152) be forbidden as well with service accounts?

Reply:
Sure, good catch